src/applications/static-pages/simple-forms/20-10207/App.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+const App = ({ formEnabled }) => {
+  if (formEnabled === undefined) {
+    return <va-loading-indicator message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>You can submit this form online or by mail.</p>
+        <a
+          className="vads-c-action-link--blue"
+          href="/supporting-forms-for-claims/request-priority-processing-form-20-10207"
```

Comment:
You should be able to use the `va-link` component here: https://design.va.gov/storybook/?path=/docs/components-va-link--default

Reply:
Disregard! I see that we don't have the [action link](https://design.va.gov/storybook/?path=/docs/components-action-link--primary) variation of this as a web component yet.

Reply:
Replaced with va-link.  Thanks for catching!